enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/ClientEvaluationService.java

```diff
@@ -141,8 +136,7 @@ private Consumer<EvaluateClientResponse> createSuccessConsumer(final IdentityVer
             final IdentityVerificationPhase phase = identityVerification.getPhase();
             if (response.isAccepted()) {
                 logger.info("Client evaluation accepted for {}", identityVerification);
-                saveInANewTransaction(status ->
-                        identityVerificationService.moveToPhaseAndStatus(identityVerification, phase, ACCEPTED, ownerId));
+                identityVerificationService.moveToPhaseAndStatus(identityVerification, phase, ACCEPTED, ownerId);
```

Comment:
Why not, but does it work? The reason for this mojo was, that the transaction processing this batch used to be `read-only`.

Reply:
I will double check this, thanks for noticing.

Reply:
It should work, because the client evaluation action is triggered by state machine which starts a transaction on background using:

```
                transactionTemplate.setPropagationBehavior(TransactionDefinition.PROPAGATION_REQUIRES_NEW);
                transactionTemplate.executeWithoutResult(status -> { ... });
```

Or from REST API call using:
```
    @Transactional
    public StateMachine<OnboardingState, OnboardingEvent> processStateMachineEvent(OwnerId ownerId, String processId, OnboardingEvent event) { ... }
```

We can review if we need additional transactions in the future, however such behaviour goes unfortunately against the locks which span only a single transaction.

Reply:
Thanks for the verification. I have realized that most probably it is a relic of client evaluation batch processing https://github.com/wultra/enrollment-server/commit/0c70f964ab0531788cd9bdee98e744a84b6bc463